src/cljs/rems/table.cljs

```diff
@@ -259,9 +267,63 @@
  (fn [db [_ table key]]
    (contains? (get-in db [::selected-rows (:id table)]) key)))
 
+(rf/reg-sub
+ ::selected-rows
+ (fn [db [_ table]]
+   (get-in db [::selected-rows (:id table)])))
+
+(rf/reg-event-db
+ ::set-selected-rows
+ (fn [db [_ table rows]]
+   (let [new-db (assoc-in db [::selected-rows (:id table)] (set (map :key rows)))]
+     (when-let [on-select (:on-select table)]
+       (on-select (get-in new-db [::selected-rows (:id table)])))
```

Comment:
Instead of looking up the selected rows from `new-db` right after putting them there, they could be saved to a local variable for reuse: `(let [selected-rows (set (map :key rows))]`

Reply:
Yes, maybe not worth the change but I did it anyway.